about.html

```diff
@@ -57,8 +59,10 @@ <h2><p>info@haitianotaku.ht</p></h2>
             <div class="about-text">
                 <h2>Haitian Otaku Global Summit 2023 Logo</h2>
                 <span class="hr"></span>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellendus excepturi unde sed deserunt sunt culpa itaque incidunt, dignissimos dolore! Officia cupiditate enim, aspernatur iste ratione iure nostrum perspiciatis veritatis ipsa?</p>
-                <!-- <img src="./images/haitianOtaku.jpeg" alt="logo" > -->
+                <p>The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to 7, July.<br>
+                After online public voting and Selection Committee’s judging process, the logo from Naresh Agrawal won the prize.<br>
+                After consultation with Mr. Agrawal, we optimized the prize-winning-logo for this year’s summit as follows:</p>
+                <img src="./images/haitianOtaku.jpeg" alt="logo" > 
```

Comment:
You're making great progress on your project! Here are a couple of suggestions to further improve it:

- [ ] As noted in the previous review, the background color in logo should be white, but it currently has a dark background. Please make sure to update the background color to match the design's specifications.

- [ ] [Optional] Additionally, it seems that the image should be a logo which is missing the required large padding and a border. Please add these elements to the logo as per the design.
![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/94bc58a8-69f0-43f3-bbfe-d71914ba93a5)

These adjustments will contribute to a more accurate and visually appealing representation of the design. Keep up the good work! 🌟📝🚀